File: articles/testing-verification-and-computational-reliability/c/reliability_audit.c
#include <stdio.h>
int score_in_range(double x){ return x >= 0.0 && x <= 100.0; }
int main(void){ printf("test_name,status\nscore_72,%s\nscore_150,%s\n", score_in_range(72.0) ? "pass" : "fail", score_in_range(150.0) ? "pass" : "fail"); return 0; }

File: articles/testing-verification-and-computational-reliability/java/src/main/java/org/contentcatalyst/algorithms/ReliabilityAudit.java
package org.contentcatalyst.algorithms;
public class ReliabilityAudit {
  static boolean scoreInRange(double x){ return x >= 0.0 && x <= 100.0; }
  public static void main(String[] args){
    System.out.println("test_name,status");
    System.out.println("score_72," + (scoreInRange(72) ? "pass" : "fail"));
    System.out.println("score_150," + (scoreInRange(150) ? "pass" : "fail"));
  }
}
